<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmGaugePanel</class>
 <widget class="QWidget" name="frmGaugePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="GaugePanel" name="gaugePanel1" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="GaugePanel" name="gaugePanel3" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="GaugePanel" name="gaugePanel4" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="GaugePanel" name="gaugePanel2" native="true"/>
      </item>
      <item row="2" column="2">
       <widget class="GaugePanel" name="gaugePanel6" native="true"/>
      </item>
      <item row="2" column="1">
       <widget class="GaugePanel" name="gaugePanel5" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="horizontalSlider3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="horizontalSlider2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="horizontalSlider1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSlider" name="horizontalSlider4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="horizontalSlider5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSlider" name="horizontalSlider6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GaugePanel</class>
   <extends>QWidget</extends>
   <header>gaugepanel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
